win/action.yml

```diff
@@ -0,0 +1,100 @@
+name: Deploy Setup Action for Windows
+description: Download direct tools
+inputs:
+  solsta_client_id:
+    description: Client ID to authenticate usage of Solid State Networks console tools
+    required: true
+    default: ""
+  solsta_client_secret:
+    description: Secret Key to authenticate usage of Solid State Networks console tools
+    required: true
+    default: ""
+  console_version:
+    description: Version of Solsta Console Tools to use
+    required: true
+    default: "6.1.2.51"
+  scripts_version:
+    description: Version of Solsta Deploy Scripts to use
+    required: true
+    default: "3.7.18"
+
+outputs:
+  result:
+    description: The state of the action, if processes completed successfully
+
+runs:
+  using: composite
+  steps:
+    - name: Create work directory
+      shell: powershell
```

Comment:
Give the above comments by @nmoinvaz and since this is always run on windows, do we need the shell in these? Or does windows default to powershell?

Reply:
My point is that you don't need 3 separate workflows, one for each platform, if you use `shell: bash`.

Reply:
Hmm... I'm not sure that's true.  It was random errors that sent me down this path to begin with.  I'll force the yaml to use the same linux script for everything and give you an example of the errors.  Off the top of my head, I only recall Windows puking on the linux "if [ ... " syntax, but I'll post details in a  sec.

Reply:
As mentioned above, it's all been consolidated now.  So it's an entirely new script to review.